docs(setup): Edited WSL setup and Local Setup sections (#48641)
freeCodeCamp/freeCodeCamp: docs/how-to-setup-freecodecamp-locally.md

| Prerequisite                                                                                  | Version | Notes                                                                                       |
| --------------------------------------------------------------------------------------------- | ------- | ------------------------------------------------------------------------------------------- |
| [Node.js](http://nodejs.org)                                                                  | `18.x`  | We use the "Active LTS" version, See [LTS Schedule](https://nodejs.org/en/about/releases/). |
| npm (comes bundled with Node)                                                                 | `8.x`   | We use the version bundled with Node.js Active LTS.                                         |

> | [MongoDB Community Server](https://docs.mongodb.com/manual/administration/install-community/) | `4.2.x` | There is no official installion/support for Ubuntu 22.04, we recommend you install with docker with the version mentioned in [Installing Docker](https://contribute.freecodecamp.org/#/how-to-setup-wsl?id=installing-docker-desktop)                                                                                          |

Reviewer: Hopefully, this changes soon. Also, the formatting for this suggestion might not be correct.

```suggestion
| [MongoDB Community Server](https://docs.mongodb.com/manual/administration/install-community/) | `4.2.x` | MongoDB has not official support for Ubuntu 22.04, you can follow the [Installing Docker Desktop](https://contribute.freecodecamp.org/#/how-to-setup-wsl?id=installing-docker-desktop) step to run MongoDB Community Server within a container |
```

> Unless you have MongoDB running in a setup different than the default, the URL stored as the `MONGOHQ_URL` value in the `.env` file should work fine. If you are using a custom configuration, modify this value as needed.
>
> If you followed along with the [Windows 10 via WSL2 Setup Guide](how-to-setup-wsl.md), then you should be able to skip this step if the MongoDB server from that guide is already running. You can confirm this by checking that you can reach `http://localhost:27017` on your local machine.

> >
> >If you are using gitpod you may skip starting mongo as it should already be running

Reviewer: I suggest we remove this in favour of adjusting this line in the `use-a-local-machine-or-gitpod` section:

> That's it, you can now skip to the 'syncing up from parent' section after you have launched a GitPod workspace.

To

> That's it, you can now skip to the [follow best-practices section](/codebase-best-practices?id=codebase-best-practices) after you have launched a Gitpod workspace.

```suggestion
```

Author: Sweet. That works cause then they would know what they can skip

Once ready, open a web browser and **visit <http://localhost:8000>**. If the app loads, sign in. Congratulations – you're all set! You now have a copy of freeCodeCamp's entire learning platform running on your local machine.

> > [!NOTE]
> > If you are using gitpod you will have to run the following command on your terminal to get your gitpod localhost url.
> > ```console
> gp url 8000
> 
> 
> 

Reviewer: This is not necessary. The ports are automatically opened, and can be found in the VSCode UI, if wanted.

```suggestion
```

Author: Thanks for the feedback. I was just giving this suggestion because on my end I was unable to see it. I was able to see the local ports but not the gitpod url. But this might be similar to what you said in previous comments where if someone has issues they can refer to the provided documentation link
